tests/handlers/test_e2e_keys.py

```diff
@@ -831,3 +837,62 @@ def test_query_devices_remote_sync(self):
                 }
             },
         )
+
+    @parameterized.expand(
+        [([],), ([{"device_id": "device_1"}, {"device_id": "device_2"}],)]
```

Comment:
It might be worth describing what the different inputs are here and why they matter.

Reply:
Good idea, thanks!